frame/dvm/src/lib.rs

```diff
@@ -111,7 +128,10 @@ decl_storage! {
 	}
 	add_extra_genesis {
 		build(|_config: &GenesisConfig| {
-			<Module<T>>::store_block();
+			<Module<T>>::store_block(false);
+
+			// Initialize the storage schema at the well known key.
+			frame_support::storage::unhashed::put::<EthereumStorageSchema>(&PALLET_ETHEREUM_SCHEMA, &EthereumStorageSchema::V1);
```

Comment:
If `V1` is setup in `on_genesis`, there could be issues with middle-flight running network(Crab&Darwinia), which means we might need extra migration for those existing networks to upgrade?

Reply:
Yes, when integrating DVM to crab, the storage entry, EthereumStorageSchema, does not exist on other networks, so you need to add this entry manually, I think migration can do this, create a new issue?

Reply:
I see, I created an issue here: https://github.com/darwinia-network/darwinia/issues/624